aws-rds-dbinstance/aws-rds-dbinstance.json

```diff
@@ -232,10 +232,29 @@
       "type": "string",
       "description": "The Active Directory directory ID to create the DB instance in. Currently, only MySQL, Microsoft SQL Server, Oracle, and PostgreSQL DB instances can be created in an Active Directory Domain."
     },
+    "DomainAuthSecretArn": {
+      "type": "string",
+      "description": "The ARN for the Secrets Manager secret that contains the credentials for the user performing the domain join."
+    },
+    "DomainDnsIps": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "description": "The IPv4 DNS IP addresses of your primary and secondary Active Directory domain controllers."
+    },
+    "DomainFqdn": {
+      "type": "string",
+      "description": "Specifies the fully qualified domain name of an Active Directory domain."
+    },
     "DomainIAMRoleName": {
       "type": "string",
       "description": "Specify the name of the IAM role to be used when making API calls to the Directory Service."
```

Comment:
Seems this one as well - https://awscli.amazonaws.com/v2/documentation/api/latest/reference/rds/modify-db-instance.html
`
    The name of the IAM role to use when making API calls to the Directory Service.

    This setting doesn’t apply to RDS Custom DB instances.
`